src/layouts/lecture/styles/layout.scss

```
@@ -38,52 +34,66 @@
   display: none !important;
 }
 
-// Responsive grid system
 .cols,
 .columns {
   display: flex;
   flex-wrap: wrap;
   margin-right: -6px;
   margin-left: -6px;
+}
 
-  &.col-gapless {
-    margin-right: 0;
-    margin-left: 0;
-
-    & > .column {
-      padding-right: 0;
-      padding-left: 0;
-    }
-  }
+.cols.col-gapless,
+.columns.col-gapless {
+  margin-right: 0;
+  margin-left: 0;
+}
 
-  &.col-oneline {
-    flex-wrap: nowrap;
-    overflow-x: auto;
-  }
+.cols.col-oneline,
+.columns.col-oneline {
+  flex-wrap: nowrap;
+  overflow-x: auto;
 }
 
 [class~='col-'],
 .column {
   flex: 1;
   max-width: 100%;
-  padding-right: 6px;
-  padding-left: 6px;
+  padding: 8px;
+}
 
-  &.col-12,
-  &.col-11,
-  &.col-10,
-  &.col-9,
-  &.col-8,
-  &.col-7,
-  &.col-6,
-  &.col-5,
-  &.col-4,
-  &.col-3,
-  &.col-2,
-  &.col-1,
-  &.col-auto {
-    flex: none;
-  }
+[class~='col-'].col-12,
+.column.col-12,
```

Won't this much same as 
```suggestion
.column.col-12,
```

---

No, `[class~='col-'].col-12` and `.column.col-12` are not the same.

`[class~='col-'].col-12` is a CSS selector that targets elements with a class that contains the word `"col-"` and also have a class of `"col-12"`. This means that it would match elements that have multiple classes, as long as one of them contains `"col-" `and the other is `"col-12"`.

On the other hand, `.column.col-12` is a selector that targets elements that have both `"column"` and `"col-12" `classes. This means that it would only match elements that have exactly those two classes and no others.

So while both selectors target elements with a `"col-12" `class, they have different conditions for the other classes that may be present.

---

Yep, i mean if these selectors map same elements in our case, then we can get rid of them, but if not let's leave it as it is

---

No, they are different. Frankly speaking, I would prefer to replace this layout system with something like 'Tailwind' because it fits better with the current needs of styling the 'mdx' format pages. But it will take few weeks.